apps/metadata-editor/src/app/records/my-org-records/my-org-records.component.ts

```diff
@@ -15,30 +15,42 @@ import { AuthService } from '@geonetwork-ui/api/repository/gn4'
   standalone: true,
   imports: [CommonModule, TranslateModule, RecordsListComponent],
 })
-export class MyOrgRecordsComponent implements OnDestroy {
-  subscriptionAuthService: Subscription
-  subscriptionOrgService: Subscription
+export class MyOrgRecordsComponent {
+  orgName: string
+  logoUrl: string
+  recordCount: number
+  userCount: number
 
   constructor(
     public searchFacade: SearchFacade,
     private authService: AuthService,
     private orgService: OrganizationsServiceInterface
   ) {
     this.searchFacade.resetSearch()
+    this.authService.user$.subscribe((user) => {
+      // this.orgName = 'Géo2France'
+      this.orgName = user?.organisation
+      this.searchByOrganisation({ name: this.orgName })
+    })
+    this.orgService.organisations$.subscribe((orgs) => {
+      const org = orgs.filter((org) => org.name === this.orgName)[0]
+      if (org) {
+        this.logoUrl = org.logoUrl?.href.toString()
+        this.recordCount = org.recordCount
+      }
+    })
 
-    this.subscriptionAuthService = this.authService.user$.subscribe((user) => {
-      this.searchByOrganisation({ name: user?.organisation })
+    this.authService.allUsers$.subscribe((users) => {
```

Comment:
Those subscribes seems to be duplicated ( with some tweeks) between and `my-org-users`. I don't know if it could  be a good thing to have a `my-org.service` with some generic methods. 

WDYT ?

Reply:
I guess that would make sense, since it's done twice the exact same way! Will try it.